cli/dataset.go

```diff
@@ -267,7 +267,7 @@ func binaryDataToJSONLines(ctx context.Context, client datapb.DataServiceClient,
 	}
 	bboxAnnotations := convertBoundingBoxes(datum.GetMetadata().GetAnnotations().GetBboxes())
 	jsonl = ImageMetadata{
-		ImagePath:                 filenameForDownload(datum.GetMetadata()),
+		ImagePath:                 filepath.Join(dst, dataDir, filenameForDownload(datum.GetMetadata())),
```

Comment:
Snuck in this other change which will allow the workflows to run without any weird modifications